runtime/src/main/java/org/corfudb/runtime/view/ObjectBuilder.java

```diff
@@ -83,31 +80,26 @@ public T open() {
             streamID = CorfuRuntime.getStreamID(streamName);
         }
 
-        if (useCompiledClass)
-        {
-            try {
-                if (options.contains(ObjectOpenOptions.NO_CACHE)) {
-                    return CorfuCompileWrapperBuilder.getWrapper(type, runtime, streamID,
-                            arguments, serializer);
-                }
-                else {
-                    ObjectsView.ObjectID<T> oid = new ObjectsView.ObjectID(streamID, type);
-                    return (T) runtime.getObjectsView().objectCache.computeIfAbsent(oid, x -> {
-                        try {
-                            return CorfuCompileWrapperBuilder.getWrapper(type, runtime, streamID,
-                                    arguments, serializer);
-                        } catch (Exception ex) {
-                            throw new RuntimeException(ex);
-                        }}
-                    );
-                }
-            } catch (Exception ex) {
-                log.error("Couldn't use compiled class for {}, using runtime instrumentation.", type);
-                throw new RuntimeException(ex);
+        try {
+            if (options.contains(ObjectOpenOptions.NO_CACHE)) {
+                return CorfuCompileWrapperBuilder.getWrapper(type, runtime, streamID,
+                        arguments, serializer);
+            } else {
+                ObjectsView.ObjectID<T> oid = new ObjectsView.ObjectID(streamID, type);
+                return (T) runtime.getObjectsView().objectCache.computeIfAbsent(oid, x -> {
+                            try {
+                                return CorfuCompileWrapperBuilder.getWrapper(type, runtime, streamID,
+                                        arguments, serializer);
+                            } catch (Exception ex) {
+                                throw new RuntimeException(ex);
+                            }
+                        }
+                );
             }
+        } catch (Exception ex) {
+            log.error("Couldn't use compiled class for {}, using runtime instrumentation.", type);
```

Comment:
I would use the message from before:
"Runtime instrumentation no longer supported and no compiled class found"